Scroller/tests/Scroller-specs.js

```diff
@@ -19,167 +20,161 @@ describe('Scroller', () => {
-
-				expect(actual).toBe(expected);
-			}
-		);
+		test('should set "aria-label" to previous scroll button in the horizontal scroll bar', () => {
+			const label = 'custom button aria label';
+			render(
+				<Scroller
+					horizontalScrollbar="visible"
+					scrollLeftAriaLabel={label}
+					verticalScrollbar="visible"
+				>
+					{contents}
+				</Scroller>
+			);
 
-		test(
-			'should set "aria-label" to next scroll button in the horizontal scroll bar',
-			() => {
-				const label = 'custom button aria label';
-				const subject = mount(
-					<Scroller
-						horizontalScrollbar="visible"
-						scrollRightAriaLabel={label}
-						verticalScrollbar="visible"
-					>
-						{contents}
-					</Scroller>
-				);
-
-				const expected = label;
-				const actual = subject.find('ScrollButton').at(3).prop('aria-label');
-
-				expect(actual).toBe(expected);
-			}
-		);
+			const previousScrollbarButtonLabel = screen.getByLabelText(label);
+			const previousScrollbarButton = screen.getByLabelText(label).parentElement;
 
-		test(
-			'should set "aria-label" to previous scroll button in the vertical scroll bar',
-			() => {
-				const label = 'custom button aria label';
-				const subject = mount(
-					<Scroller
-						horizontalScrollbar="visible"
-						verticalScrollbar="visible"
-						scrollUpAriaLabel={label}
-					>
-						{contents}
-					</Scroller>
-				);
-
-				const expected = label;
-				const actual = subject.find('ScrollButton').at(0).prop('aria-label');
-
-				expect(actual).toBe(expected);
-			}
-		);
+			expect(previousScrollbarButtonLabel).toBeInTheDocument();
+			expect(previousScrollbarButton).toHaveClass('horizontal');
+		});
 
-		test(
-			'should set "aria-label" to next scroll button in the vertical scroll bar',
-			() => {
-				const label = 'custom button aria label';
-				const subject = mount(
-					<Scroller
-						horizontalScrollbar="visible"
-						verticalScrollbar="visible"
-						scrollDownAriaLabel={label}
-					>
-						{contents}
-					</Scroller>
-				);
-
-				const expected = label;
-				const actual = subject.find('ScrollButton').at(1).prop('aria-label');
-
-				expect(actual).toBe(expected);
-			}
-		);
+		test('should set "aria-label" to next scroll button in the horizontal scroll bar', () => {
+			const label = 'custom button aria label';
+			render(
+				<Scroller
+					horizontalScrollbar="visible"
+					scrollRightAriaLabel={label}
+					verticalScrollbar="visible"
+				>
+					{contents}
+				</Scroller>
+			);
+
+			const nextScrollbarButtonLabel = screen.getByLabelText(label);
+			const nextScrollbarButton = screen.getByLabelText(label).parentElement;
+
+			expect(nextScrollbarButtonLabel).toBeInTheDocument();
+			expect(nextScrollbarButton).toHaveClass('horizontal');
+		});
+
+		test('should set "aria-label" to previous scroll button in the vertical scroll bar', () => {
+			const label = 'custom button aria label';
+			render(
+				<Scroller
```

Comment:
You can put props on the same line if you like.
<Scroller horizontalScrollbar="visible" scrollUpAriaLabel={label} verticalScrollbar="visible">
	{contents}
</Scroller>